.github/workflows/get-translations.yml

```diff
@@ -31,17 +37,42 @@ jobs:
       - name: Fetch latest dev
         run: git fetch origin dev
 
-      - name: Get translations
-        run: npx ts-node -O '{"module":"commonjs"}' ./src/scripts/crowdin/translations/getTranslations.ts
-        env:
-          CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY }}
-          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
-
       - name: Authenticate GitHub CLI
         run: |
           echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
 
+      # Build translations
+      - name: Build Crowdin project
+        id: build-crowdin
+        run: |
+          OUTPUT=$(npx ts-node -O '{"module":"commonjs"}' ./src/scripts/crowdin/translations/triggerBuild.ts)
+          echo "$OUTPUT"
```

Comment:
This syntax echo's the `console.log` statements made from inside `triggerBuild.ts`. Console log statements matching a particular pattern of `::set-output name=key_name::value` can be extracted in a future step.

In `triggerBuild.ts` we're console logging `::set-output name=buildId::${id}` which allows us to get the `buildId` variable next.